### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by samatharhay**
@ishank-arora I've added an outline of my summary, Sorry that its not all that flushed out. I will try to have a second draft completed by noon tomorrow, so if you don't mind waiting til then to start on your review that would be really appreciated. Thanks

### CODE REVIEW

**Comment by ishank-arora** [comment]
> @ishank-arora I've added an outline of my summary, Sorry that its not all that flushed out. I will try to have a second draft completed by noon tomorrow, so if you don't mind waiting til then to start on your review that would be really appreciated. Thanks

@samatharhay no worries, take your time, I won't start on my review until around Friday afternoon at the earliest

File: _summaries/vaswani2017attention_1.md [hunk 1]
```diff
@@ -1,19 +1,88 @@
...
+All sub-layers have residual connections and normalization, like the sub-layers in the encoder.
+The self-attention layer has added masking to prevent positions from attending to subsequent positions, which helps ensure that predictions for position i can depend only on known outputs for positions before it.
+
+
+The Transformer uses two different types of attention: Scaled Dot-Product Attention and Multi-Head Attention.
+
+![Attention](vaswani2017attention_1b.png)
+
+Scaled Dot-Product Attention
+* Takes as input queries and keys of dimension dk and values of dimension dv.
```
Comment (ishank-arora) [inline _summaries/vaswani2017attention_1.md:37]: Please consider making this a bit clearer, maybe like so: "Takes queries (Q) and keys (K) of dimension $$d_k$$, and values (V) of dimension $$d_v$$ as inputs."

File: _summaries/vaswani2017attention_1.md [hunk 2]
```diff
@@ -1,19 +1,88 @@
...
+ 
+![SDPA](vaswani2017attention_1d.png)
+
+* Identical to Dot-product attention except for the scaling factor
```
Comment (ishank-arora) [inline _summaries/vaswani2017attention_1.md:41]: I think you should specify that the scaling factor is 1/(sqrt(d_k))

**Review by ishank-arora — COMMENTED** [review]
Overall, great summary. Succinct, extracts the relevant information about the groundbreaking Transformer model from the paper to communicate to your audience. I've made some minor comments here and there for you to review.  I also think you can add a couple more things. 

Firstly, as part of the summary, I think it could be beneficial to see some advantages and disadvantages of the proposed Transformer model in their own dedicated section. 

Secondly, it would also be great to see is a quick explanation of what the terms attention and self-attention mean. I think those would greatly enhance the quality of your summary, and allow readers to better appreciate the groundbreaking nature of what the authors are proposing here

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**